### PR TITLE
Designfront

### PR DESCRIPTION
**Pull request by apostolclaudia**

### CODE REVIEW

File: frontend/src/pages/AddProduct.vue [hunk 1]
```diff
@@ -1,14 +1,88 @@
 <template>
   <q-page class="flex flex-center">
     <!-- <img alt="Quasar logo" src="leek2.png" style="height: 200px" /> -->
```
Comment (CatalinAtanase) [inline frontend/src/pages/AddProduct.vue]: sterge asta

File: frontend/src/pages/AddProduct.vue [hunk 2]
```diff
@@ -1,14 +1,88 @@
...
+    const age = ref(null);
+    const accept = ref(false);
+
+    return {
+      name,
+      age,
+      accept,
+
+      onSubmit() {
+        if (accept.value !== true) {
```
Comment (CatalinAtanase) [inline frontend/src/pages/AddProduct.vue:62]: moara mama ce e mizeria asta

Comment (CatalinAtanase) [reply frontend/src/pages/AddProduct.vue:62]: if(accept) ???

File: frontend/src/pages/Cooking.vue [hunk 3]
```diff
@@ -1,7 +1,54 @@
 <template>
-  <q-page class="flex flex-center">
+  <q-page class="column items-center">
     <!-- <img alt="Quasar logo" src="leek2.png" style="height: 200px" /> -->
```
Comment (CatalinAtanase) [inline frontend/src/pages/Cooking.vue]: sterge asta

File: frontend/src/pages/Friends.vue [hunk 4]
```diff
@@ -1,7 +1,44 @@
...
+      type="search"
+      hint="Search new friends"
+    >
+      <template v-slot:append>
+        <q-icon name="search" />
+      </template>
+    </q-input>
+
+    <q-card flat bordered class="card-class text-center" color="secondary">
+      <!-- <div class="stroke-menu rotate-90"></div> -->
```
Comment (CatalinAtanase) [inline frontend/src/pages/Friends.vue:16]: sterge asta

File: frontend/src/pages/Index.vue [hunk 5]
```diff
@@ -1,6 +1,13 @@
 <template>
   <q-page class="column justify-center items-center">
-    <img alt="LeekIt logo" src="leek2.png" style="width: 200px" />
+    <img alt="LeekIt logo" src="leek2.png" style="width: 300px" />
+    <q-card flat bordered class="card-class text-center" color="secondary">
+      <!-- <div class="stroke-menu rotate-90"></div> -->
```
Comment (CatalinAtanase) [inline frontend/src/pages/Index.vue]: sterge asta

**Review by CatalinAtanase — CHANGES_REQUESTED** [review]

File: frontend/src/pages/AddProduct.vue (hunk 2 above)
Comment (apostolclaudia) [reply frontend/src/pages/AddProduct.vue:62]: NICOLE E DE VINA NU EU

**Review by CatalinAtanase — APPROVED** [review]

**Pull request merged**